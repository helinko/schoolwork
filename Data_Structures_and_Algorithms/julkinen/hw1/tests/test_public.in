A Captain 2233 2250 Gold James T. Kirk
A Commander 2230 2250 Blue Spock
A Commander 2222 2241 Red Montgomery Scott
A MedicalOfficer 2227 2251 Blue Leonard McCoy
A Lieutenant 2237 2255 Gold Hikaru Sulu
A Lieutenant 2239 2257 Red Nyota Uhura
A Lieutenant 2232 2250 Gold Lee Kelso
A Ensign 2245 2263 Gold Pavel Chekov
F 1
F 2
F 3
F 4
F 5
F 6
F 7
F 8
X young
X old
Q
 
