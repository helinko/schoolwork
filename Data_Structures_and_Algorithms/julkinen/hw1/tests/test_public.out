> A Captain 2233 2250 Gold James T. Kirk
> A Commander 2230 2250 Blue Spock
> A Commander 2222 2241 Red Montgomery Scott
> A MedicalOfficer 2227 2251 Blue Leonard McCoy
> A Lieutenant 2237 2255 Gold Hikaru Sulu
> A Lieutenant 2239 2257 Red Nyota Uhura
> A Lieutenant 2232 2250 Gold Lee Kelso
> A Ensign 2245 2263 Gold Pavel Chekov
> F 1
Montgomery Scott Commander 2222 2241 Red
> F 2
Spock Commander 2230 2250 Blue
> F 3
Lee Kelso Lieutenant 2232 2250 Gold
> F 4
James T. Kirk Captain 2233 2250 Gold
> F 5
Leonard McCoy MedicalOfficer 2227 2251 Blue
> F 6
Hikaru Sulu Lieutenant 2237 2255 Gold
> F 7
Nyota Uhura Lieutenant 2239 2257 Red
> F 8
Pavel Chekov Ensign 2245 2263 Gold
> X young
Pavel Chekov Ensign 2245 2263 Gold
> X old
Montgomery Scott Commander 2222 2241 Red
> Q
